packit.yaml

```diff
@@ -13,6 +13,9 @@ srpm_build_deps:
   - glib2-devel
   - make
   - systemd-devel
+files_to_sync:
+  - packit.yaml
+  - cockpit.spec
```

Comment:
@allisonkarlitskaya wants this gone. So let's file a packit issue and link it from here.

Reply:
+1, please let us know what would be a better solution for you as we recently did an overhaul to the files_to_sync functionality.

Reply:
When I added that to umockdev, I hit an unlucky time -- this was broken in packit back then, see https://github.com/packit/packit/issues/1525. But this got fixed in the meantime. I sent https://github.com/martinpitt/umockdev/pull/184 to drop this, and will test it there. If/when successful, I'll drop it here as well.

Reply:
Worked, dropped.